src/main/java/pl/javastart/task/Truck.java

```diff
@@ -0,0 +1,35 @@
+package pl.javastart.task;
+
+public class Truck extends Car {
+    private double cargoWeight;
+
+    public Truck(String name, double fuelTankCapacity, double fuelConsumption, boolean airConditioning, double cargoWeight) {
+        super(name, fuelTankCapacity, fuelConsumption, airConditioning);
+        this.cargoWeight = cargoWeight;
+    }
+
+    public double getCargoWeight() {
+        return cargoWeight;
+    }
+
+    public void setCargoWeight(double cargoWeight) {
+        this.cargoWeight = cargoWeight;
+    }
+
+    @Override
+    public double getFuelConsumption() {
+        double resoult = super.getFuelConsumption();
```

Comment:
Nie wywołuj metody z klasy narzędnej, bo jeżeli zmieni się tam wartość, to ta metoda nie będzie działała poprawnie